ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/legaladvisor/CCDHearingDurationType.java

```diff
@@ -4,6 +4,7 @@
     ONE_DAY,
     FOUR_HOURS,
     THREE_HOURS,
+    TWO_AND_HALF_HOURS,
```

Comment:
do unit/integration tests need to be updated/added?

Reply:
I'm not sure

Reply:
Have a look at DrawJudgeOrderCallbackHandlerTest.java I think it is used there. Perhaps you could decide looking at this unit test. Its a small change though, so up to you. Also not sure who's responsibility to update integration tests is.

Reply:
I don't think we need to make a new unit test for testing the new field. I could modify one of the unit/integration tests to see if the new field can be used.
Specifically changing this:
<img width="605" alt="ProposedChange1" src="https://user-images.githubusercontent.com/94063348/149503601-0827a3a5-5e6c-49e6-8407-5ceb9ce4b709.PNG">
to this:
<img width="655" alt="ProposedChange2" src="https://user-images.githubusercontent.com/94063348/149503665-61bfb39b-438c-4ead-b390-1e83533a2cfd.PNG">
I have to modify it anyway to move the new enum to the bottom (I'm doing this just in case it messes with any database).

Reply:
Okie. cool. I don't see the change you made. perhaps you did not push your changes?

Reply:
Should be pushed now